distribution/tools/server-cli/src/main/java/org/elasticsearch/server/cli/ServerProcess.java

```diff
@@ -216,6 +216,7 @@ private static Process createProcess(
         command.addAll(jvmOptions);
         command.add("--module-path");
         command.add(esHome.resolve("lib").toString());
+        command.add("--add-modules=jdk.net");
```

Comment:
maybe add a comment here that this is a very special circumstance, more explicit modules should not be added here

Reply:
Done.

Reply:
Done.